src/context/card-form-context.tsx

```diff
@@ -133,10 +176,19 @@ const cardFormReducer = (state, action) => {
   }
 };
 
-const CardFormProvider = ({ children }) => {
+interface Props {
+  children: React.ReactNode;
+}
+
+interface TValue {
+  state: TState;
+  dispatch: (action: TAction) => void;
+}
+
+const CardFormProvider = ({ children }: Props) => {
```

Comment:
이렇게 하면 굳이 children 타입을 직접 정의해줄 필요 없답니다 :)

```suggestion
interface TValue {
  state: TState;
  dispatch: (action: TAction) => void;
}

const CardFormProvider = ({
  children,
}: React.PropsWithChildren<Record<string, React.ReactElement>>) => {
````